wire-ios/Wire-iOS Tests/ConversationImagesViewControllerTests.swift

```diff
@@ -48,7 +49,7 @@ final class ConversationImagesViewControllerTests: CoreDataSnapshotTestCase {
     override func setUp() {
         super.setUp()
         SelfUser.setupMockSelfUser()
-
+        userSession = UserSessionMock()
```

Comment:
Shouldn't we pass in the mock self user to the user session?

Reply:
We're leaving it as it is for now as discussed.